webapp/src/pages/ResponsePage/ResponsePage.css

```diff
@@ -29,12 +25,19 @@ button {
     margin-bottom: 10px;
 }
 
- .table-wrapper .btn-primary {
-    background:hsl(208deg 84% 59%);
+label {
+    font-size: 12px;
+    color: grey;
+    margin: 0 1px;
+}
+
+
+.table-wrapper.response-page .btn-primary {
```

Comment:
### Let's keep global styles in App.css and only if they've been tested across the entire app. This one in particular significantly impacts the readability of the application form (so please change to to be specific to this page)

I believe I have attributed this specifically to this page please see class "response-page" ?

Reply:
The comment was referring to the "label" one above, not this one...